CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
+- Add feature to initialize with native options ([#221](https://github.com/getsentry/sentry-kotlin-multiplatform/pull/221))
```

Comment:
```suggestion
- Allow initializing SentryKMP SDK with native options ([#221](https://github.com/getsentry/sentry-kotlin-multiplatform/pull/221))
```
I feel like "feature" is excessive here, since it's under the "Features" section already. Also, can we add a code snippet here to showcase how users can provide platform options?

Reply:
yup added code snippet as well